network/wsPeer_test.go

```diff
@@ -104,6 +104,11 @@ func TestAtomicVariablesAlignment(t *testing.T) {
 	require.True(t, (unsafe.Offsetof(p.lastPacketTime)%8) == 0)
 	require.True(t, (unsafe.Offsetof(p.intermittentOutgoingMessageEnqueueTime)%8) == 0)
 	require.True(t, (unsafe.Offsetof(p.duplicateFilterCount)%8) == 0)
+	require.True(t, (unsafe.Offsetof(p.txMessageCount)%8) == 0)
```

Comment:
this is great and why don't we have this for everything we `sync/atomic` access?

https://github.com/algorand/go-algorand/issues/5001

Reply:
looks like can be done as a custom linter similar to part test, by using [fieldalignment](https://cs.opensource.google/go/x/tools/+/refs/tags/v0.5.0:go/analysis/passes/fieldalignment/fieldalignment.go;l=58;bpv=0;bpt=0) as an example